jetty-server/src/main/java/org/eclipse/jetty/server/handler/InetAccessHandler.java

```diff
@@ -140,6 +170,8 @@ public void dump(Appendable out, String indent) throws IOException
     {
         dumpObjects(out, indent,
             DumpableCollection.from("included",_set.getIncluded()),
-            DumpableCollection.from("excluded",_set.getExcluded()));
+            DumpableCollection.from("included",_set.getIncluded()),
```

Comment:
excluded not duplicate of included

Reply:
NB this is how to do the dump in InetAccessHandler if you want to fix it!

Reply:
done